src/components/calcite-button/calcite-button.tsx

```diff
@@ -108,6 +108,22 @@ export class CalciteButton {
       .reduce((acc, { name, value }) => ({ ...acc, [name]: value }), {});
   }
 
+  // if a button has an associated form, create a proxy button and click it
+  private handleClick = (e: Event) => {
+    const requestedForm = this.el.getAttribute("form");
+    const targetForm = requestedForm
+      ? document.querySelector(`#${requestedForm}`)
```

Comment:
would `document.getElementById` be better here since its an id?

Reply:
Yep will update!